frontend/src/utils/utils.js

```diff
@@ -0,0 +1,9 @@
+// enums 
+
+// signal types
+export const SIGNALS = {
+    OPEN: "open",
+    LIKED: "liked",
+    READ : "read"
+};
```

Comment:
Instead of having a SIGNALS object and another one for WEIGHTS, another option is just to use one object like this, or similar
```
export const SIGNALS = {
    OPEN: { type: "open", weight: 4 },
    LIKED: { type: "liked", weight: 3 },
    READ : { type: "read", weight: 3 },
};
```

Reply:
Oh my bad! I didn't notice that. You're all good then!